RELEASE.md

```diff
@@ -14,6 +14,7 @@ Please follow the established format:
 - Change route name from `runsList` to `experiment-tracking`. (#820)
 - Update feature flag description to remind the user of the need for page refresh to apply settings. (#821)
 - Fix experiment tracking not showing run details bug on Windows. (#809)
+- Fix flowchart not showing on intiial load for static data inputs. (#840)
```

Comment:
Oh forgot this: don't we usually reference the PR number, not the issue number? So this would be `#843` and not `#840`.